### PR TITLE
SUR-249 - Editor input component

### PR DESCRIPTION
**Pull request by jaieds**
### Description

- Please check out the documentation [Readme.md](https://github.com/brainstormforce/force-ui/blob/75b9378fcb4e4ca7a26120667af36814ce63a0d3/src/components/editor-input/readme.md)

### Checklist:

-   [x] My code is tested
-   [ ] My code passes the PHPCS tests
-   [x] I've created the npm build.
-   [x] My code follows accessibility standards 
-   [x] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [x] I've included developer documentation 
-   [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by JelenaTakac** [comment]
Is it allowed to set multiple instances of the same option? 👉 https://bsf.d.pr/i/kqgpEB

**Review by JelenaTakac — COMMENTED** [review]
Is it allowed to set multiple instances of the same option? 👉 https://bsf.d.pr/i/kqgpEB

**Comment by jaieds** [comment]
> Is it allowed to set multiple instances of the same option? 👉 https://bsf.d.pr/i/kqgpEB

@JelenaTakac, Yes.

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -32,10 +32,14 @@
     "react-dom": "^18.3.1"
   },
   "dependencies": {
+    "@emotion/is-prop-valid": "^1.3.0",
```
[the remark this answers is not in the document]
Comment (jaieds) [reply package.json:35]: Yes. We need it for the `framer-motion` library. They are using this library to validate props.

**Review by vrundakansara — APPROVED** [review]

**Review by JelenaTakac — APPROVED** [review]
Tested.

**Pull request merged**